app/common/state/ledgerState.js

```diff
@@ -356,6 +356,7 @@ const ledgerState = {
   setRecoveryStatus: (state, status) => {
     state = validateState(state)
     const date = new Date().getTime()
+    state = state.setIn(['about', 'preferences', 'recoveryInProgress'], false)
```

Comment:
should this be `state.setIn([....], !status)` or will it always be false on setRecoveryStatus?

Reply:
the `setRecoveryStatus` method inside ledger state is called when the conclusion of the recovery has occurred, (success/fail) or when it is reset. In both cases `recoveryInProgress` should be set to false as the attempted recovery is over.

Reply:
please use `aboutPreferencesState.setPreferencesProp` here as well

Reply:
please move this function `setRecoveryStatus` into `aboutPreferencesState` and use `setPreferencesProp` for all 3 props. Thank you

Reply:
@NejcZdovc refactoring pushed